New package: BashMacros v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: BashMacros
- Repository: https://github.com/obsidianjulua/BashMacros.jl
- Created by: @obsidianjulua
- Version: v0.1.0
- Commit: ae3d02e0bb6f00fb5b9038b31e78123ef9818d62
- Reviewed by: @obsidianjulua
- Reference: https://github.com/obsidianjulua/BashMacros.jl/commit/ae3d02e0bb6f00fb5b9038b31e78123ef9818d62#commitcomment-168072403
- Description: Julia and Bash, full macros and startup
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6356927e14ed9e759b7a2f1f751a385b91106953424af4c948b063bf7760235f29f4b2d094875b81c896f9bb5eefdc46bfe6c37d48eb934a150b96960bda079a80f2da40de3ecd6a516005cd48067ec16e8b9411e3e1feb08ced6f52feb4b640d7451176585fd747a7da944e5d3118b92770c1f9f88d49f978006c402638820f461e644003a511f5fe9ade4d3295a34efccc03618150b7ba41a05e0ef4e81bc8610c0210d181a204aa53f0b3d62bee2a4 -->

goerz commented:
This seems like something that contains at least some amount of LLM hallucinations. In the README,

````
> **Note:** Ensure `BashMacros.jl` and its dependencies (`eBash.jl`, etc.) are accessible (e.g., by ensuring your project directory is in the `JULIA_LOAD_PATH`).

```julia
# In your ~/.julia/config/startup.jl file:
# ... (contents from the provided startup.jl file)
include("BashMacros.jl") # Custom Bash integration framework
using .BashMacros         # Access functions/macros like @bash, bash_full
using Revise
# ... other utilities
```
````

is pretty much nonsense.

Please do not submit LLM-generated packages where you have not manually double-checked every line.

JuliaRegistrator committed:
New package: BashMacros v0.1.0

UUID: 05aca6c6-4af4-4687-bf5f-b81b4e60d6a9
Repo: https://github.com/obsidianjulua/BashMacros.jl.git
Tree: 5589af1af70a5d09e7d2a8ab4ecea547c82816d5

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

obsidianjulua commented:
Hmm. If you haven't installed the pkg to your main julia env. You have to include("") and using. Anyone that knows julia can switch it to just using BashMacros, since your referencing my example startup.jl and the way I use it on my specific instance and saying its nonsense but its exactly how using the startup.jl works with a unregistered pkg. Could you explain anything other than you think its llm generated and make negative comments but not provide real information on what a problem is other than maybe you dont like the way it looks, did I forget a space or something. To me what you say is nonsense, do you even use bash or linux to understand what its doing?

obsidianjulua commented:
I did finally get type conversations to work between bash execution and julia meaning true data processing between the two. Opening up distributed possibilities and keeping strict format. This was very hard but I did remove the startup.jl requirements and implemented the logic directly between bashrc and julia. Sorry for the original but I managed to fix type conversation literally right after this. Your right about etiquette for programming in julia, and im working on better docs that I make myself rather that taking the easy road with the llm because the code can work perfect but die hard on bad docs. So I completely understand
